__tests__/pet.test.js

```diff
@@ -0,0 +1,232 @@
+const Pet = require('../src/pet');
+
+const minFitness = 0;
+const maxFitness = 10;
+
+const minAge = 0;
+const maxAge = 30;
+
+const minHunger = 0;
+const maxHunger = 10;
+
+const yearIncrement = 1; // adds one year to age
+const hungerIncrement = 5; // adds 5+ to hunger level when grows up
+
+const errorMsg = 'Your pet is no longer alive :(';
+const walkMsg = 'I need a walk'
+const hungryMsg = 'I am hungry'
+const greatMsg = 'I feel great'
+
+describe('constructor', () => {
+  let pet;
+  beforeEach(() => {
+   pet = new Pet('Jessie');
+  });
+    it('returns an object', () => {
+      expect(new Pet('Jessie')).toBeInstanceOf(Object);
+    });
+    it('sets the name property', () => {
```

Comment:
Adding a clear line between each `it` block would make it a bit easier to read
```suggestion

    it('sets the name property', () => {
```